airflow/models/dagrun.py

```diff
@@ -125,14 +125,16 @@ def refresh_from_db(self, session=None):
 
     @staticmethod
     @provide_session
-    def find(dag_id=None, run_id=None, execution_date=None,
+    def find(dag_id=None, dag_ids=None, run_id=None, execution_date=None,
```

Comment:
Do we need a separate argument for this? we can make dag_id to support list & str both

Reply:
Yeah, that's a fairly common pattern in our other methods.

Reply:
I will change it during the next rebase.

Reply:
Can we add type annotations here as well?

Reply:
In this PR, I want to focus only on necessary changes. I want to add type-hint in separate PR, but type-hint in core are problematic. They cause cyclical imports.